src/pages/admin/presentation/[id].tsx

```diff
@@ -0,0 +1,197 @@
+import { useRouter } from 'next/router';
+import { useContext, useEffect, useState } from 'react';
+import StarterKitContext from '../../../context/starterKit';
+import UserNeedsMoreRights from '../../../components/UserNeedsMoreRights';
+import * as Yup from 'yup';
+import usePlatform from '../../../hooks/usePlatform';
+import { useProvider, useSigner } from 'wagmi';
+import { useChainId } from '../../../hooks/useChainId';
+import { postToIPFS } from '../../../utils/ipfs';
+import { createMultiStepsTransactionToast, showErrorTransactionToast } from '../../../utils/toast';
+import { Field, Form, Formik } from 'formik';
+import { ethers } from 'ethers';
+import { useConfig } from '../../../hooks/useConfig';
+import TalentLayerPlatformID from '../../../contracts/ABI/TalentLayerPlatformID.json';
+import { useWeb3Modal } from '@web3modal/react';
+import SubmitButton from '../../../components/Form/SubmitButton';
+import { Container } from '../../../components/newlayout/container';
+import Steps from '../../../components/Steps';
+import Loading from '../../../components/Loading';
+
+interface IFormValues {
+  name: string;
+  about: string;
+  website: string;
+  video_url: string;
+  image_url: string;
+}
+
+const validationSchema = Yup.object({
+  // nothing required
+});
+
+function AdminPresentation({ callback }: { callback?: () => void }) {
```

Comment:
Better to not have everything directly the page code. We can extract the form in a component.